### ISSUE TITLE
osxpmem "Cannot open file /dev/pmem: No such file or directory" message error is confusing

### ISSUE THREAD

**Issue by jipegit** [Velocidex/c-aff4#87]
The error message "E Cannot open file /dev/pmem: No such file or directory" can be confusing, especially because it is the only feedback given to the user in non-verbose or non-debug mode.

In fact it looks like something bad happened during the dump process whereas it is just telling the user /dev/pmem does not exist yet and osxpmem needs to load its driver to use it.

I think the message should be more explicit.

**Comment by scudette** [comment]
Yes you are right - it is confusing. We typically try to unload the driver and reload it because it might e.g. be an old version of the driver (although the driver is almost never updated anyway). I am not sure if this is a good strategy - perhaps if we find the driver already there we might try to use it anyway? Of course it could always be something completely different which is also called /dev/pmem by accident.

**Comment by jipegit** [comment]
I thing it is a good strategy. From a general standpoint It looks like a more robust solution to always use the version included in the osxpmem binary ran by the user. I would not return an error to the user if /dev/pmem does not exist though, of maybe only include it in debug mode.
